uw-frame-components/portal/features/partials/announcement.html

```diff
@@ -21,7 +21,7 @@
     <md-menu-item layout="row" layout-align="space-between center">
       <span>Announcements</span>
       <a href="features" ng-click="markAnnouncementsSeen(true)" aria-label="see all announcements" class="link__see-all">
```

Comment:
Does this do the same as the notifications link? This marks announcements as seen, but I would have thought the notification link would take me to the notification page